app/views/branches/_expression_answers.html.erb

```diff
@@ -0,0 +1,16 @@
+<div class="" data-expression-index="<%= expression_index %>" data-component-type="<%= component_type %>">
+  <%= f.select :operators, @expression.operators,
+    { include_blank: t('branches.select_operator') },
+    {
+      class: 'govuk-select',
+      name: "branch[conditionals_attributes][#{conditional_index}][expressions_attributes][#{expression_index}][operator]",
+      id: "branch_conditionals_attributes_#{conditional_index}_expressions_attributes_#{expression_index}_operator"
+    }  %>
+  <%= f.select :answers, @expression.answers,
+    { include_blank: t('branches.select_answer') },
+    {
+      class: 'govuk-select',
+      name: "branch[conditionals_attributes][#{conditional_index}][expressions_attributes][#{expression_index}][answer]",
+      id: "branch_conditionals_attributes_#{conditional_index}_expressions_attributes_#{expression_index}_answer"
+    } %>
+</div>
```

Comment:
I am not sure if we can use `child_index` or not. 🤔 
Even if we can't, can we dry the naming moving into a helper method? Wdyt?

Reply:
sounds good - can use a helper method to dry it up